public/datasets/zoo.json

```diff
@@ -0,0 +1,113 @@
+{
+  "fields": [
+    {
+      "type": "categorical",
+      "id": "Animal Name",
+      "description": "Name of the animal"
+    },
+    {
+      "type": "categorical",
+      "id": "Hair",
+      "description": "Does the animal have hair?"
+    },
+    {
+      "type": "categorical",
+      "id": "Feathers",
+      "description": "Does the animal have feathers?"
+    },
+    {
+      "type": "categorical",
+      "id": "Eggs",
+      "description": "Does the animal lay eggs?"
+    },
+    {
+      "type": "categorical",
+      "id": "Milk",
+      "description": "Does the animal produce milk?"
+    },
+    {
+      "type": "categorical",
+      "id": "Airborne",
+      "description": "Does the animal fly?"
+    },
+    {
+      "type": "categorical",
+      "id": "Aquatic",
+      "description": "Does the animal live in water?"
+    },
+    {
+      "type": "categorical",
+      "id": "Predator",
+      "description": "Is the animal a predator?"
+    },
+    {
+      "type": "categorical",
+      "id": "Toothed",
+      "description": "Does the animal have teeth?"
+    },
+    {
+      "type": "categorical",
+      "id": "Backbone",
+      "description": "Does the animal have a vertebrae and backbone?"
```

Comment:
This wording seemed a little off to me. I think of the "backbone" as comprised of series of individual bones, vertebrae. Maybe "Does the animal have vertebrae?" or "Does the animal have a backbone?"